Fails to install based on devDependencies

On `npm i --save imager.jsx` I get the following error:
```
Error: Cannot find module 'lodash.debounce' from '/Users/newtriks/src/projects/sayansho/sayansho-co-pilot/node_modules/imager.jsx'
```
The postinstall process is run and the devDependencies are not actually installed. I can confirm that installing all the imager.jsx devDependencies locally and then installing imager.jsx works.

--- a/README.md
+++ b/README.md
@@ -16,8 +16,10 @@ npm i --save imager.jsx
 If you want to generate a static artifact, clone the repository and run the following commands:
 
 ```bash
+git clone https://github.com/oncletom/Imager.jsx.git imager.jsx && cd imager.jsx
 npm install
 npm run build
+open demo/index.html
 ```
 
 # Usage

--- a/demo/index.html
+++ b/demo/index.html
@@ -44,7 +44,6 @@
 <script type="text/jsx">
   /** @jsx React.DOM */
   var React = require('react');
-  var debounce = require('lodash.debounce');
   var Imager = imager.jsx;
 
   // Imager.js configuration

--- a/package.json
+++ b/package.json
@@ -9,9 +9,8 @@
   },
   "scripts": {
     "build": "npm run build-vendor && npm run build-bundle",
-    "build-vendor": "browserify -r react -r lodash.debounce -o dist/vendor.js",
+    "build-vendor": "browserify -r react -o dist/vendor.js",
     "build-bundle": "browserify -x react src/imager.jsx -s imager.jsx -o dist/bundle.js",
-    "postinstall": "if [[ $NODE_ENV != 'production' ]]; then npm run build; fi",
     "test": "echo \"Error: no test specified\" && exit 1",
     "watch": "watchify -x react src/imager.jsx -s imager.jsx -o dist/bundle.js"
   },
@@ -42,7 +41,6 @@
   },
   "devDependencies": {
     "browserify": "^6.3.2",
-    "lodash.debounce": "^2.4.1",
     "reactify": "^0.17.1",
     "watchify": "^2.1.1"
   }

